airflow/ui/src/pages/DagsList/DagsFilters.tsx

```diff
@@ -51,6 +57,21 @@ export const DagsFilters = () => {
       [pagination, searchParams, setSearchParams, setTableURLState, sorting],
     );
 
+  const handleStateChange: React.MouseEventHandler<HTMLButtonElement> =
+    useCallback(
+      ({ target }) => {
+        const { value } = target as HTMLButtonElement;
+
+        if (value === "all") {
+          searchParams.delete(STATE_PARAM);
+        } else {
+          searchParams.set(STATE_PARAM, value);
+        }
+        setSearchParams(searchParams);
```

Comment:
For the 'pause' we reset the display to 1st page. But here we don't do it.

Is there a reason for having an inconsistency of behavior between the filters ? I would expect to also reset the page to the first 1.

Reply:
Makes sense. Updated the code. Thanks.